Project1/preview-cards/index.html

```diff
@@ -5,28 +5,46 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0"> <!-- displays site properly based on user's device -->
 
   <link rel="icon" type="image/png" sizes="32x32" href="./images/favicon-32x32.png">
-  
+  <link rel="stylesheet" href="style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Big+Shoulders+Display:wght@700&family=Lexend+Deca&display=swap" rel="stylesheet">
+
   <title>3-column preview card component</title>
 
-  <!-- Feel free to remove these styles or customise in your own stylesheet 👍 -->
-  <style>
-    .attribution { font-size: 11px; text-align: center; }
-    .attribution a { color: hsl(228, 45%, 44%); }
-  </style>
+
 </head>
 <body>
+  <main>
+    <div class="content">
+      <div class="card">
+      <div class="container-1">
+        <img src="/Project1/preview-cards/images/icon-sedans.svg" alt="icon of a sedan car">
```

Comment:
tak będzie działać http://mediologia.pl/katalogi-i-pliki-linux/wzgledna-i-absolutna-sciezka-dostepu
```suggestion
                <img src="images/icon-sedans.svg" alt="icon of a sedan car">

```